src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,14 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userEmail = new UserService();
```

Comment:
it's better to change naming 
```suggestion
        UserService userService = new UserService();
```